src/bindings/c/include/openvino/c/ov_rank.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is a header file for ov_shape C API
+ *
+ * @file ov_rank.h
+ */
+
+#pragma once
+
+#include "openvino/c/ov_common.h"
+typedef struct ov_rank ov_rank_t;
```

Comment:
```suggestion
typedef struct ov_dimension ov_rank_t;
```
And ov_dimension should be defined as a public structure (user should be able to see the structure members).

Reply:
sure, new C style definition will be applied.